ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -87,18 +84,26 @@ def handler(request, context):
         self.assertEqual(span.name, "")
         self.assertIs(span.kind, trace.SpanKind.SERVER)
 
+        self.assertEqual(
```

Comment:
Maybe an utility function to encapsulate these assertions (parameterized with the root module? or name/version separately) this would also make sense.

Reply:
Nice suggestion. Done!